packages/map/src/circle/index.jsx

```diff
@@ -2,7 +2,7 @@ import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 
 import Layer from '../layer';
-import { useChildrenBounds } from '../map/hooks';
+import { useAddLayerBoundsToMap } from '../map/hooks';
```

Comment:
just `addBounds`?

Reply:
it's hook. needs to be prefixed with use. Renamed to useAddLayerBounds